hnn_core/tests/test_compare_hnn.py

```diff
@@ -124,15 +123,10 @@ def test_mpi_failure():
     # this MPI paramter will cause a MPI job with more than one process to fail
     environ["OMPI_MCA_btl"] = "self"
 
-    # this is a bummer, but pytest on linux does something w/ file descriptors
-    if 'linux' in sys.platform:
```

Comment:
Since all output goes to stdout now, we can just use `redirect_stdout`, which works correctly in Linux.